def fatalError(self, exception):
    'Handle a non-recoverable error.'
    raise exception
